.gitlab/lassen-jobs.yml

```diff
@@ -23,7 +23,7 @@ clang_10_0_1_mpi (build and test on lassen):
 
 gcc_8_3_1:
   variables:
-    SPEC: "+fortran%gcc@8.3.1"
+    SPEC: "+fortran%gcc@8.3.1 ^cmake@3.14.5"
```

Comment:
How does this cause CMake 3.18.0 to be used?  Other than the CHANGELOG, I don't see any other changes being made.

Reply:
I think it's a Spack spec syntax thing. It's hard to find the right documentation on it (imho) but it seems to be something that lets you handle dependencies (https://spack.readthedocs.io/en/latest/environments.html#speclists-as-constraints)

Reply:
But I'm confused why this one change will make the CI then always use cmake 3.18 for Lassen - what about the other jobs in the file?

Reply:
Lol I think Marty and I have the same confusion (it's late mkay)

Reply:
I think @davidbeckingsale intent was not to use cmake@3.18.0 for this job specifically, but set it as a default for the other jobs. The config change happens in the Radiuss-CI submodule.